server/app/attestation-service-ctl/readme.md

```diff
@@ -0,0 +1,129 @@
+```SHELL
+./attestation-service-ctl get-policy --tee <TEE> [--addr <ADDR>] [--output <PATH>]
+```
+- tee: Specify the target TEE name.
+- addr(optional): Designate the attestation server's `management-sock` address. Default is `127.0.0.1:3001`.
+- output(optional): Output file path to write the policy. Default is `./Policy.rego`.
+
+### set-policy
+
+Set the TEE-specific evaluation `Policy(.rego)` according to the content of `POLICY`.
+```SHELL
+./attestation-service-ctl set-policy --tee <TEE> --policy <POLICY> [--addr <ADDR>]
+```
+- tee: Specify the target TEE name.
+- policy: The path of local `Policy(.rego)` which will be upload to Attestation Server and saved in the AS user's workdir: `/opt/attestation-service/<TEE>/`.
+- addr(optional): Designate the Attestation Server's `management-sock` address. Default is `127.0.0.1:3001`.
+
+### get-reference-data
+
+Get the TEE-specific `Reference Data(.json)` from Attestation Server.
+```SHELL
+./attestation-service-ctl get-reference-data --tee <TEE> [--addr <ADDR>] [--output <PATH>]
+```
+- tee: Specify the target TEE name.
+- addr(optional): The Attestation Server's `management-sock` address. Default is `127.0.0.1:3001`.
+- output(optional): Output file path to write the reference data. Default is `./Reference_data.json`.
+
+### set-reference-data
+
+Set the TEE-specific evaluation `Reference Data(.json)` according to the content of `REFERENCE_DATA`.
+```SHELL
+./attestation-service-ctl set-reference-data --tee <TEE> --reference-data <REFERENCE_DATA> [--addr <ADDR>]
+```
+- tee: Specify the target TEE name.
+- reference-data: The path of local `Reference Data(.json)` which will be upload to Attestation Server and saved in the AS user's workdir: `/opt/attestation-service/<TEE>/`.
+- addr(optional): Designate the Attestation Server's `management-sock` address. Default is `127.0.0.1:3001`.
+
+### restore-default-policy
+
+Restore the Attestation Server's TEE-specific `Policy(.rego)` to default.
+```SHELL
+./attestation-service-ctl restore-default-policy --tee <TEE> [--addr <ADDR>]
+```
+- tee: Specify the target TEE name.
+- addr(optional): The Attestation Server's `management-sock` address. Default is `127.0.0.1:3001`.
+
+### restore-default-reference-data
+
+Restore the Attestation Server's TEE-specific `Reference Data(.json)` to default.
+```SHELL
+./attestation-service-ctl restore-default-reference-data --tee <TEE> [--addr <ADDR>]
+```
+- tee: Specify the target TEE name.
+- addr(optional): The Attestation Server's `management-sock` address. Default is `127.0.0.1:3001`.
+
+### test-attestation
+
+Test Attestation Server's `attestation` function with the input `EVIDENCE` and print the attestation result to stdout.
+```SHELL
+./attestation-service-ctl test-attestation [--evidence <EVIDENCE>] [--addr <ADDR>]
+```
+- evidence(optional): The evidence which is evaluated by Attestation Server. Default is an embedded `sample` TEE's evidence.
```

Comment:
When not using the `sample` evidence, what's the format for `<EVIDENCE>` ?

Reply:
It depends on which verifier the user wants to test. For example, if the user want to test a TDX verifier, he can prepare a evidence of the format defined by a TDX verifier.

Reply:
Right. So should we pass a file path containing that evidence? It'd be easier for complex evidence formats like TDX or SEV.